src/main/java/org/example/services/Racing.java

```diff
@@ -2,12 +2,12 @@
 
 import org.example.entities.Pair;
 
-import java.util.List;
-
 public interface Racing {
-
-    public List<Pair> creator();
-
+    
+    Pair randomizer();
```

Comment:
looks like you use only one method outside service, in this case just remove all other methods from interface